### PR TITLE
citesummary: adds cite summary view

### PR DESCRIPTION
**Pull request by eamonnmag**
- ADD Adds citesummary endpoint.
- TODO: Needs data endpoint available @jacquerie.

Signed-off-by: Eamonn Maguire eamonnmag@gmail.com

### CODE REVIEW

File: inspirehep/modules/theme/bundles.py [hunk 1]
```diff
@@ -106,3 +100,20 @@
...
+    output="gen/citesummary.%(version)s.js",
+    npm={
+        "inspire-citesummary-js": "^0.0.4"
+    }
+)
+
+citesummary_css = NpmBundle(
+    "node_modules/dc/dc.min.css",
+    "node_modules/inspire-citesummary-js/examples/style.css",
+    filters='requirejs',
```
Comment (jmartinm) [inline inspirehep/modules/theme/bundles.py]: for the css?

File: inspirehep/modules/theme/templates/inspirehep_theme/citations/citation_summary.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#}
+
+{%- extends "inspirehep_theme/page.html" -%}
```
Comment (jmartinm) [inline inspirehep/modules/theme/templates/inspirehep_theme/citations/citation_summary.html]: `{%- extends config.BASE_TEMPLATE -%}` would be better

File: inspirehep/modules/theme/bundles.py [hunk 3]
```diff
@@ -71,12 +71,6 @@
     }
 )
 
-detailedjs = NpmBundle(
```
Comment (eamonnmag) [inline inspirehep/modules/theme/bundles.py]: yeah, it was duplicated for some reason.

File: inspirehep/modules/theme/bundles.py (hunk 1 above)
Comment (eamonnmag) [inline inspirehep/modules/theme/bundles.py]: You mean the use of NPMBundle?

Comment (eamonnmag) [inline inspirehep/modules/theme/bundles.py]: ah, filters. indeed!

**Comment by jacquerie** [comment]
I will take care of merging this this afternoon :+1:

**Comment by jacquerie** [comment]
Superseded by #1699

**Pull request closed without merging**